src/plugins/notebook/components/NotebookMenuSwitcher.vue

```diff
@@ -50,66 +57,63 @@ export default {
     },
     mounted() {
         validateNotebookStorageObject();
-        this.getDefaultNotebookObject();
 
         this.notebookSnapshot = new Snapshot(this.openmct);
         this.setDefaultNotebookStatus();
     },
     methods: {
-        getDefaultNotebookObject() {
-            const defaultNotebook = getDefaultNotebook();
+        getPreviewObjectLink() {
+            let path = '#/browse/';
+            path += this.objectPath
+                .map(p => this.openmct.objects.makeKeyString(p.identifier))
+                .reverse()
+                .join('/')
+            ;
+            path += `?view=${this.currentView.key}`;
```

Comment:
In the next line we get all of the current parameters, and filter out the view parameter. It seems like it might be simpler to get all of the parameters, **set** the view parameter, then convert them all to a string?
eg.
```JavaScript
const urlParams = this.openmct.router.getParams();
urlParams.view = this.currentView.key;

path += "?" + Object.entries(thing)
    .map(([key, value]) => `${key}=${value}`)
    .join('&');
```